Tests/CMakeLists.txt

```diff
@@ -93,7 +93,7 @@ add_custom_target(tests-cleanup ${CMAKE_COMMAND} -E remove testrunner.xml)
 set_target_properties(tests-cleanup PROPERTIES FOLDER Testing)
 
 if(OGS_USE_PETSC)
-    set(TEST_FILTER_MPI --gtest_filter=-MPITest_Math.*)
+    set(TEST_FILTER_MPI --gtest_filter=-MPITest_Math.*:MPITest_NumLib.*)
```

Comment:
Would setting the filter to `-MPITest*` still work?
It would make two disjoint sets for `tests` and `tests_mpi` targets.